### ISSUE TITLE
Microphone Not Responding in VS Code Speech

### ISSUE THREAD

**Issue by 17life-Max** [microsoft/vscode#232782]
I’m experiencing an issue with VS Code’s speech-to-text feature. When I activate the microphone, it doesn’t respond, but the microphone works fine with other applications outside of VS Code.
The microphone permission for VS Code is enabled, and it was working fine yesterday, but today it suddenly stopped working.

I checked the log and found this error message:
```
Speech provider error in speech to text session: Runtime error: Failed to initialize the embedded speech engine. Check the paths and permissions of library and model files, and ensure that the model version is supported and the license is valid. [Runtime error: License error. Loading the model requires a valid license.]
```
How can I resolve this issue? I’m using macOS Sequoia 15.1, VS Code 1.95.1, VS Code Speech v0.12.0

**Comment by tobiasbueschel** [comment]
@17life-Max I'm experiencing the same issue after updating to macOS 15.1 yesterday, hence, I'm not sure if it's related to the OS update or a change on the VS Code Speech side?

**Comment by tarek-eg** [comment]
> [@17life-Max](https://github.com/17life-Max) I'm experiencing the same issue since yesterday! I believe it's related to the VS Code Speech update to v0.12.0 that was rolled out ~19 hours ago.
> 
> When I downgrade to v0.10.0 everything works again. You can install a specific version like this:
> 
> ![Image](https://github.com/user-attachments/assets/fa2bfe1d-7c3c-4da6-a601-6aba882ef0cd)
> 
> Hence, I think v0.12.0 introduced some change that caused the above issue. Not sure if it now actually requires a different copilot license or if it's a bug?
> 
> ![Image](https://github.com/user-attachments/assets/8a8c1593-192e-483d-8a95-e2170ba432da)

Thanks, I had the same issue and fixed it with downgrading.

**Comment by bpasero** [comment]
Sorry for that, preparing release `0.12.1` to address that.

**Comment by bpasero** [comment]
This is now fixed with `0.12.1`, sorry for that.
